backend/open_api_spec.yml

```diff
@@ -0,0 +1,757 @@
+openapi: 3.0.0
+info:
+  version: "1.0.0"
+  title: Chan Zuckerburg Initiative cellxgene Data Portal Curator API
+servers:
+paths:
+  /curation/auth/key:
+    post:
+      summary: Generate the curator's API key -- for use with Data Portal UI
+      description: >-
+        Generate and return a curator API key for the user. If there is already a key associated with the
+        user's auth profile, replace it with the new key.
+      security:
+        - cxguserCookie: []
+      responses:
+        "201":
+          description: Created
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  key:
+                    type: string
+                  id:
+                    type: string
+        "401":
+          $ref: "#/components/responses/401"
+        "400":
+          $ref: "#/components/responses/400"
+    get:
+      summary: Fetch the identifier of the curator's active API key -- for use with Data Portal UI
+      security:
+        - cxguserCookie: []
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: string
+        "401":
+          $ref: "#/components/responses/401"
+        "404":
+          $ref: "#/components/responses/404"
+    delete:
+      summary: Delete the curator's API key -- for use with Data Portal UI
+      security:
+        - cxguserCookie: []
+      responses:
+        "202":
+          $ref: "#/components/responses/202"
+        "401":
+          $ref: "#/components/responses/401"
+
+  /curation/auth/token:
+    post:
+      summary: Generate an access token
+      description: >-
+        Returns a bearer access token that must be passed as a parameter to requests that
+        require authorization such as creating a new Collection.
+      parameters:
+        - in: header
+          name: Authorization
+          schema:
+            type: string
+          required: true
```

Comment:
^^ @Bento007 

cc @brianraymor

Reply:
If this is the right format, I'll add to the other endpoints as well (for the token)

Reply:
Authorization would not be **required** for other endpoints if only public access was desired?